### PR TITLE
Fix boolean handling in mssql source

### PR DESCRIPTION
**Pull request by sherifnada**
## What
The mssql source connector treated MSSQL `BIT` fields (used as booleans) as `int`s in the generated catalog. This causes normalization to fail in the destination because they don't expect to receive `false` and `true` as int values. This PR fixes this bug.

### CODE REVIEW

File: airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/MssqlSourceTest.java [hunk 1]
```diff
@@ -53,6 +53,7 @@
 import java.sql.SQLException;
 import java.util.Set;
 import java.util.stream.Collectors;
+import javax.swing.*;
```
Comment (cgardens) [inline airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/MssqlSourceTest.java:56]: not `.*`?

**Review by michel-tricot — APPROVED** [review]
Does that solve Jins' issue?

**Review by cgardens — APPROVED** [review]

File: airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/MssqlSourceTest.java (hunk 1 above)
Comment (sherifnada) [reply airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/MssqlSourceTest.java:56]: removed

**Comment by sherifnada** [comment]
@michel-tricot yes this fixes Jins' isssue

**Pull request merged**